dspace-api/src/main/java/org/dspace/app/mediafilter/ImageMagickThumbnailFilter.java

```diff
@@ -132,12 +136,22 @@ public static File getImageFile(File f, int page) throws IOException, Interrupte
     	f2.deleteOnExit();
     	ConvertCmd cmd = new ConvertCmd();
 		IMOperation op = new IMOperation();
+		Info imageInfo = new Info(f.getAbsolutePath(),true);
+		// I'd rather set these in the class but can't figure out how to access them here if I do!!!
```

Comment:
Since the values of the config settings contain a path to the ghostscript install, I think you will need to continue setting this inside of the config file.